packages/terra-tag/src/Tag.module.scss

```diff
@@ -56,7 +56,11 @@
       color: var(--terra-tag-focus-color, #0059a8);
       text-decoration: var(--terra-tag-focus-text-decoration);
     }
-    
+
+    &.has-underline {
+      text-decoration: underline;
```

Comment:
we can remove the text decoration from hover class of tag ( line no. 70 )
```css
text-decoration: var(--terra-tag-hover-text-decoration, underline);
```

Reply:
Are we sure that this has-underline class will be set during a hover event?  That would be the only way it could be removed, correct?